setup.cfg

```diff
@@ -31,7 +31,7 @@ packages=find:
 
 install_requires =
     py-solc
-    web3>=5.0.0,<5.2
+    web3>=5.0.0,<=5.6.0
```

Comment:
why does it not work with >5.6?

Reply:
I don't know, 5.7.0 does not exist, and it did not work with 5.6.0 before the change. So we could imagine that it does not work with >5.6 for the same reason. I could do also have `<5.7`